server/api/swagger.yaml

```diff
@@ -812,6 +812,33 @@ paths:
           description: The service
           schema:
             type: object
+  /services/{service}/asgs:
+    x-swagger-router-controller: servicesController
+    get:
+      operationId: getASGsByService
+      summary: Get the ASGs to which a service is deployed.
+      tags:
+        - Service
+      parameters:
+        - name: service
+          in: path
+          type: string
+          required: true
+        - name: environment
+          in: query
+          type: string
+          required: true
+        - name: slice
```

Comment:
An enumeration of valid stings for a slice (blue | green | none) is defined in a couple of places in the swagger doc. Suggest extracting this as its own definition.